src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -201,10 +201,14 @@ private static ISymbol MapSymbol(ISymbol symbol, ITypeSymbol type)
                 type = semanticModel.GetTypeInfo(bindableParent, cancellationToken).Type;
 
                 declaredSymbol = MapSymbol(semanticFacts.GetDeclaredSymbol(semanticModel, token, cancellationToken), type);
-                allSymbols = semanticModel.GetSymbolInfo(bindableParent, cancellationToken)
-                                              .GetBestOrAllSymbols()
-                                              .WhereAsArray(s => !s.Equals(declaredSymbol))
-                                              .SelectAsArray(s => MapSymbol(s, type));
+
+                var skipSymbolInfoLookup = declaredSymbol != null && declaredSymbol.Kind == SymbolKind.RangeVariable;
+                allSymbols = skipSymbolInfoLookup
+                    ? ImmutableArray<ISymbol>.Empty
+                    : semanticFacts.GetSymbolInfo(semanticModel, bindableParent, token, cancellationToken)
+                                   .GetBestOrAllSymbols()
```

Comment:
:bulb: To avoid indenting more than necessary, consider one of the following approaches:

1. Simply don't align on the `.`
2. Move the first `.` to the next line, so all of the `.` characters are aligned but still at a 4-space indent